moninom1/FreeRTOS
# Tcp/ip endpoint demo changes

Demo changes for adding EndPoint changes

Description
GetAddressConfiguration function updated to GetEndPointConfiguration in main.c

Test Steps
Normal Demo run steps

Related Issue
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
